geomstats/geometry/riemannian_metric.py

```diff
@@ -453,6 +453,25 @@ def normal_basis(self, basis, base_point=None):
 
         return gs.einsum("i, ikl->ikl", 1.0 / gs.sqrt(norms), basis)
 
+    def covariant_riemannian_tensor(self, base_point):
+        r"""Compute purely covariant version of Riemannian tensor at base_point.
```

Comment:
Thanks, could you also add the formula for it, i.e. R_{ijkl} = g_{ll'}R_{ijk}^l' ?

Reply:
Yes will do! As for the testing thing I thought I had resolved this but turns out I haven't yes, thanks!